pyproject.toml

```diff
@@ -65,7 +65,7 @@ optional = true
 tox = "^3.26"
 py-sr25519-bindings = "^0.1.5"
 pylint = "==2.15.5"
-cosmpy = ">=0.6.0,<0.7.0"
+cosmpy = ">=0.5.1,<0.7.0"
```

Comment:
the new release of cosmpy with dependencies improved is in it's way.

so in a while can be updated to cosmpy>=0.6.1,<0.7

Reply:
This is already done now (CosmPy 0.6.1 is out and addresses the jsonschema deps). 
Please revert this change.

Reply:
At some point I had trouble locally to install cosmpy 0.6.0, that's why I have reverted to a previous version.